.github/workflows/ci.yaml

```diff
@@ -127,14 +129,14 @@ jobs:
         cp dist/ca.pem "${scratch}/ca.pem"
 
         echo "##[group]Build"
-          cat hack/boilerplate.ytt.txt > "${scratch}/config/cartogrpaher-conventions.yaml"
-          ko resolve -B -f dist/cartogrpaher-conventions.yaml >> "${scratch}/config/cartogrpaher-conventions.yaml"
-          kbld -f "${scratch}/config/cartogrpaher-conventions.yaml" --imgpkg-lock-output "${scratch}/.imgpkg/images.yml" > /dev/null
+          cat hack/boilerplate.ytt.txt > "${scratch}/config/cartographer-conventions.yaml"
+          ko resolve -B -f dist/cartographer-conventions.yaml >> "${scratch}/config/cartographer-conventions.yaml"
+          kbld -f "${scratch}/config/cartographer-conventions.yaml" --imgpkg-lock-output "${scratch}/.imgpkg/images.yml" > /dev/null
         echo "##[endgroup]"
 
         echo "##[group]Create bundle"
           imgpkg push -f "${scratch}" -b "${BUNDLE}/controller"
```

Comment:
I noticed at the end we end up with

```
./bundle/
├── .imgpkg
│   └── images.yml
├── CHANGELOG.md
├── LICENSE
├── bundle.values.yaml
├── bundle.yaml
├── ca-overlay.yaml
├── ca.pem
└── config
    └── cartographer-conventions.yaml
```

but I couldn't figure out what `ca.pem` (empty, in the bundle) is used for - is there just because it's part of `dist/` which is referred to in the docs?

Reply:
I don't think `ca.pem` needs to be in the bundle. We use that file as a default target for local installs (see below). I'll open a separate PR to remove it.

From the readme
```
  kapp deploy -n cartographer-system -a conventions \
      -f <( \
        ko resolve -f <( \
          ytt \
            -f dist/cartographer-conventions.yaml \
            -f dist/ca-overlay.yaml \
            --data-value-file ca_cert_data=${CA_DATA:-dist/ca.pem} \
          ) \
      )
```

Reply:
done